eip-0024.md

```diff
@@ -0,0 +1,23 @@
+# Auction contract
+
+* Author: anon_real
+* Status: Proposed
+* Created: 25-Oct-2021
+* License: CC0
+* Forking: not needed 
+
+## Motivation 
+With the discovery that we can easily use spent boxes in contracts ([see here](https://www.ergoforum.org/t/ergoscript-design-patterns/222/23?u=anon_real)), some new features are introduced for artworks and can be extended furthur in the future. This EIP is an extension of [Asset standard](eip-0004.md) and tents to standardize artwrok issuance in particular.
+
+
+## Design
+There are two important boxes in the issuance process of an artwork.
+- The issuance box, which follows [Asset standard](eip-0004.md) depending on the type of artwork.
+- The issuer box, which is the first input of the transaction that is issuing the artwork. In particular, the box with the same ID as the artwork's token ID.
+
+Now we will discuss why the issuer box is important and define a standard for it.
+
+The issuer box is important because it has the same ID as the artwork's token ID and we can include it (although it is spent) in our transactions either in registers or as context data. This means that we can prove/verify certain features that an artwork may have using this box. The following is the current standard for this box - which may be extended in the future.
+
+- R4 of this box is an Int, showing 1000 * royalty percentage of the artwork. e.g, 20 for 2% royalty. If R4 of the issuer box of an artowrk is empty or non-positive, then royalty is considered to be 0%.
```

Comment:
maybe use base points, like in finance, so that 20, means 0.2%.

Reply:
Ah, I wish I knew this before. Changing this requires some tricky changes in the AH code as well which I'd like to avoid if possible. If you think that should be done, I'll do it for sure though.

Reply:
AH app definitely serves are a reference implementation for this EIP. At the same time, once accepted it may be used in other apps, at which point it is better this app doesn't have such issues.
So I can approve the EIP, but leaving it for you to decide about this issue.